major update to be able to run all  model version

florianhartig opened the pull request.

JohannesOberpriller committed:
major update to be able to run all  model version

JohannesOberpriller committed:
fixes bugs of whitespaces in the ins file and removes unncessary outcommented parameters

JohannesOberpriller commented:
I will push some improvements soon.. Maybe, there should be some code review and merge afterwards. Otherwise people will not recognice that this is no longer limited to a specific LPJ-GUESS version

JohannesOberpriller committed:
Update README.md

Update with actual workflow and placement of the files required to run the model
